backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -464,7 +462,6 @@ public void Remove(Parcel parcel)
                 this.Context.ParcelBuildings.Remove(b);
                 this.Context.BuildingEvaluations.RemoveRange(b.Building.Evaluations);
                 this.Context.BuildingFiscals.RemoveRange(b.Building.Fiscals);
-                this.Context.Buildings.Remove(b.Building);
```

Comment:
Same here, if a parcel is removed, it shouldn't just delete all of the associated buildings (it should just delete the parcel->building relationship.